qb/test/with.test.ts

```diff
@@ -5,15 +5,15 @@ test("simple repeated expression", () => {
   const numbers = e.set(e.int64(1), e.int32(2), e.int16(3));
 
   expect(e.select(e.plus(numbers, numbers)).toEdgeQL()).toEqual(`WITH
-  __withVar_0 := ({ <std::int64>1, <std::int32>2, <std::int16>3 })
+  __withVar_0 := ({ 1, <std::int32>2, <std::int16>3 })
 SELECT ((__withVar_0 + __withVar_0))`);
 });
 
 test("simple expression with alias", () => {
   const numbers = e.set(e.int64(1), e.int32(2), e.int16(3));
 
   expect(e.select(e.plus(numbers, e.alias(numbers))).toEdgeQL()).toEqual(`WITH
-  __withVar_0 := ({ <std::int64>1, <std::int32>2, <std::int16>3 }),
+  __withVar_0 := ({ 1, <std::int32>2, <std::int16>3 }),
```

Comment:
Yeah, this is the reason why I think we should mainly have functional tests

Reply:
Definitely. More fun to write too. 🤙🤙